### DISCUSSION TITLE
schema模式支持表单默认不校验吗？

### DISCUSSION THREAD

**Discussion by happyElina** [alibaba/formily#1775] [Q&A 2.x] [answered]
请问schema模式支持默认不校验，等到我需要submit的时候再校验吗？在https://react.formilyjs.org/api/shared/schema 这里没有看到相关的参数~

**Comment by janryWang** [comment, marked as answer]
triggerType配置none

**Reply by happyElina** [reply]
在schema 对应的属性下  和  x-validator 下设置triggerType:'none' 都没有起作用呢

**Reply by janryWang** [reply]
什么不起作用，把复现demo放出来

**Reply by happyElina** [reply]
`const schema = {
  type: "object",
  properties: {
    required_1: {
      triggerType: "none",
      name: "required_1",
      title: "必填",
      type: "string",
      required: true,
      "x-decorator": "FormItem",
      "x-component": "Input",
      "x-validator": {
        triggerType: "none"
      }
    }
  }
};`
是这样传参数 triggerType: "none"吗？我这样传之后，输入的时候还是会校验

**Reply by happyElina** [reply]
https://codesandbox.io/s/shy-river-xue3d?file=/src/App.js
不知道为啥在这里面就报错跑不通，不过可以看到我的代码了。。

**Reply by janryWang** [reply]
你要控制required的triggerType的话，是需要把required放到x-validator里的，x-validator:{required:true,triggerType:"none"}

**Reply by happyElina** [reply]
好的，多谢~
